webserver/script/CostBenefitAnalysis/UploadToPostgres.py

```diff
@@ -30,550 +27,235 @@ def __init__(
 
-        # cost benefit result related 
-        self.cost_benefit_year_len = 0
         self.uncontrolled_cost_benefit_result_dict = {}
-        self.controlled_cost_benefit_result_dict = {}
+        self.uncontrolled_electricity_supply_cost_list = []
+        self.uncontrolled_avoided_gasoline_gallons = []
 
         with open(settings.BASE_DIR[:-3] + 'script/CostBenefitAnalysis/cases/BaseCase_{0}_uncontrolled_load/results/annual_results.csv'.format(county)) as csv_file:
             csv_reader = csv.reader(csv_file, delimiter=',')
             for row in csv_reader:
-                self.cost_benefit_year_len = len(row) - 1
-                self.uncontrolled_cost_benefit_result_dict[row[0]] = []
-                for i in range(1, len(row)):
-                    self.uncontrolled_cost_benefit_result_dict[row[0]].append(row[i])
-                if "EV sales as " in row[0]:
-                    fill_up_value = [row[1], row[2]]
-                    for j in range(5):
-                        self.uncontrolled_cost_benefit_result_dict[row[0]].append(fill_up_value[0])
-                        self.uncontrolled_cost_benefit_result_dict[row[0]].append(fill_up_value[1])
 
-        with open(settings.BASE_DIR[:-3] + 'script/CostBenefitAnalysis/cases/BaseCase_{0}_e19controlled_load/results/annual_results.csv'.format(county)) as csv_file:
-            csv_reader = csv.reader(csv_file, delimiter=',')
-            for row in csv_reader:
-                self.controlled_cost_benefit_result_dict[row[0]] = []
-                for i in range(1, len(row)):
-                    self.controlled_cost_benefit_result_dict[row[0]].append(row[i])
-                if "EV sales as " in row[0]:
-                    fill_up_value = [row[1], row[2]]
-                    for j in range(5):
-                        self.controlled_cost_benefit_result_dict[row[0]].append(fill_up_value[0])
-                        self.controlled_cost_benefit_result_dict[row[0]].append(fill_up_value[1])
-
-        # emission result related 
-        self.emission_year_len = 0
-        self.uncontrolled_emission_result_dict = {}
-        self.controlled_emission_result_dict = {}
+                # Calculate uncontrolled Electricity Supply Cost and store to uncontrolled cost benefit results
+                # Electricity Supply Cost (per year) = Energy Supply Cost (per year) + Capacity Cost (per year) + Distribution Cost (per year) + Transmission Costs (per year) + GHG Cost (per year)
+                if row[0] == 'Energy Supply Cost' or row[0] == 'Capacity Cost' or row[0] == 'Distribution Cost' or row[0] == 'Transmission Cost' or row[0] == 'GHG Cost':
+                    for idx, item in enumerate(row[1:]):
+                        if len(self.uncontrolled_electricity_supply_cost_list) > idx:
+                            self.uncontrolled_electricity_supply_cost_list[
+                                idx] = self.uncontrolled_electricity_supply_cost_list[idx] + float(item)
```

Comment:
are you using a linter for python? These line breaks, here and below, after `[]` should be removed. They are very confusing.

Reply:
Yes I used PEP 8, will fix it!